suite-native/module-onboarding/src/screens/TrackBalancesScreen.tsx

```diff
@@ -1,36 +1,75 @@
 import { useNavigation } from '@react-navigation/native';
 
+import { useIsUsbDeviceConnectFeatureEnabled } from '@suite-native/feature-flags';
 import {
     OnboardingStackParamList,
     OnboardingStackRoutes,
     StackNavigationProps,
 } from '@suite-native/navigation';
 import { Box } from '@suite-native/atoms';
+import { useTranslate } from '@suite-native/intl';
 
 import { OnboardingFooter } from '../components/OnboardingFooter';
 import { OnboardingScreen } from '../components/OnboardingScreen';
 import { GraphSvg } from '../components/GraphSvg';
 
 export const TrackBalancesScreen = () => {
+    const { translate } = useTranslate();
+
+    const { isUsbDeviceConnectFeatureEnabled } = useIsUsbDeviceConnectFeatureEnabled();
+
     const navigation =
         useNavigation<
             StackNavigationProps<OnboardingStackParamList, OnboardingStackRoutes.TrackBalances>
         >();
 
-    const handleRedirect = () => {
-        navigation.navigate(OnboardingStackRoutes.AboutReceiveCoinsFeature);
-    };
+    const TrackBalancesScreenWithDevice = () => (
```

Comment:
Same here, have these components separate and only use them in `TrackBalancesScreen` based on the FF

Reply:
done in 829658806